web/src/components/button.js

```diff
@@ -24,18 +25,18 @@ export default function Button({
         isSecondary ? secondaryStyle : ''
     }${isSecondaryDark ? secondaryDarkStyle : ''}`;
 
-    if (to) {
+    if (internalLink) {
         return (
-            <Link to={to} className={finalStyles}>
+            <Link to={`${link}`} className={finalStyles}>
```

Comment:
I'd add the leading `/` here that way we aren't relying on the user to get the string just right
```
to={`/${link}`}
```